consensus/pacemaker/pacemaker_debug.go

```diff
@@ -0,0 +1,31 @@
+package pacemaker
+
+import typesCons "github.com/pokt-network/pocket/consensus/types"
+
+// This Pacemaker interface is only used for development & debugging purposes.
+type PacemakerDebug interface {
+	SetManualMode(bool)
+	IsManualMode() bool
+	ForceNextView()
+}
+
+type paceMakerDebug struct {
+	manualMode                bool
+	debugTimeBetweenStepsMsec uint64
+
+	// IMPROVE: Consider renaming to `previousRoundQC`
```

Comment:
Can we do this as long as we're updating the code

Reply:
agreed, done.